installer.sh

```diff
@@ -0,0 +1,442 @@
+    output "+--------------------------------------------------+" "heading"
+
+    output "\nChecking environment" "heading"
+}
+
+function outro() {
+    output ""
+    output "+--------------------------------------------------+" "success"
+    output "|                                                  |" "success"
+    output "|   Platform.sh has been installed successfully.   |" "success"
+    output "|                                                  |" "success"
+    output "+--------------------------------------------------+" "success"
+
+    output "\nWhat's next?" "heading"
+
+    output "  Get started with: platform welcome" "output"
+
+    output "\nUseful links:" "heading"
+    output "  CLI introduction: https://docs.platform.sh/get-started/introduction.html#cli"
+
+    if [ ! -z "$footer_notes" ]; then
+        output "\nWarning during installation:" "heading"
+        output "$footer_notes" "warning"
+    fi
+
+    output "\nThank you for using Platform.sh!"
+}
+
+function add_footer_note() {
+    for var in "$@"; do
+        if [ ! -z "$footer_notes" ]; then
+            footer_notes="${footer_notes}\n${var}"
+        else
+            footer_notes="${footer_notes}${var}"
+        fi
+    done
+}
+
+function indent() {
+    OLDIFS=$IFS
+    IFS=$'\n'
+    while read -r data; do
+        line=$(echo "   | ${data}"|sed $'s/\r/\r   | /g'|sed $'s/\x1B\[[0-9;]\{1,\}[A-Za-z]//g')
+        output "$line" "comment"
+    done
+    IFS=$OLDIFS
+}
+
+# Check that cURL is installed
+function check_curl() {
+    if command -v curl >/dev/null 2>&1; then
+        output "  [*] cURL is installed" "success"
+        if gh auth status >/dev/null 2>&1; then
+            GITHUB_TOKEN="$(gh auth token)"
+            if ! github_curl https://api.github.com/repos/platformsh/cli/releases/latest >/dev/null 2>&1; then
+                GITHUB_TOKEN=""
+            else
+                output "  [*] Using GitHub auth from the gh CLI" "success"
+            fi
+        elif [ ! -z "${GITHUB_TOKEN}" ]; then
+            if ! github_curl https://api.github.com/repos/platformsh/cli/releases/latest >/dev/null 2>&1; then
+                GITHUB_TOKEN=""
+            else
+                output "  [*] Using GitHub auth from the GITHUB_TOKEN env variable" "success"
+            fi
+        fi
+    else
+        output "  [ ] ERROR: cURL is required for installation" "error"
+        exit_with_error
+    fi
+}
+
+# Check that Gzip is installed
+function check_gzip() {
+    if command -v gzip >/dev/null 2>&1; then
+        output "  [*] Gzip is installed" "success"
+    else
+        output "  [ ] ERROR: Gzip is required for installation" "error"
+        exit_with_error
+    fi
+}
+
+function check_version() {
+    if [ -z "${VERSION}" ]; then
+        version=$(github_curl https://api.github.com/repos/platformsh/cli/releases/latest | grep "tag_name" | cut -d \" -f 4)
+        output "  [*] No version specified, using latest ($version)" "success"
```

Comment:
```suggestion
        output "  [*] Installing latest version: $version" "success"
```
to look more positive